runtime/gc_glue_java/MarkingSchemeRootClearer.cpp

```diff
@@ -331,7 +331,7 @@ MM_MarkingSchemeRootClearer::scanContinuationObjects(MM_EnvironmentBase *env)
 							} else {
 								/* object was not previously marked */
 								gcEnv->_markJavaStats._continuationCleared += 1;
-								VM_VMHelpers::cleanupContinuationObject((J9VMThread *)env->getLanguageVMThread(), object);
+								VM_VMHelpers::cleanupContinuationObject((J9VMThread *)env->getLanguageVMThread(), object, (MM_GCExtensions::verify_continuation_list == _extensions->continuationListOption));
```

Comment:
is this really necessary? wouldn't shouldScanContinuationObjects return false (since the lists are empty)?